### PR TITLE
Clean grunt completion

### PR DESCRIPTION
**Pull request by tbhaxor**
## Description

Simplify the completion file with evaluation of `grunt --completion=bash` and fixed linting

## Motivation and Context

Help clean bashit #1696 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] If I have added a new file, I also added it to ``clean_files.txt`` and formatted it using ``lint_clean_files.sh``.
- [ ] I have added tests to cover my changes, and all the new and existing tests pass.

### CODE REVIEW

File: completion/available/grunt.completion.bash [hunk 1]
```diff
@@ -1,75 +1,6 @@
...
-  local compls="$(echo "$gruntinfo" | awk '/Available tasks: / {$1=$2=""; print $0}')"
-  # Only add -- or - options if the user has started typing -
-  [[ "$cur" == -* ]] && compls="$compls $opts"
-  # Tell complete what stuff to show.
-  COMPREPLY=($(compgen -W "$compls" -- "$cur"))
-}
-
-complete -o default -F _grunt_completions grunt
+if _command_exists grunt; then
+	eval "$(grunt --completion=bash)"
```
Comment (NoahGorny) [inline completion/available/grunt.completion.bash:5]: I get "Specified grunt shell completion  rules for bash not found"

Comment (tbhaxor) [reply completion/available/grunt.completion.bash:5]: That's strange. I just tried and it is working.

Comment (tbhaxor) [reply completion/available/grunt.completion.bash:5]: Is it on macOS or Linux?

Comment (tbhaxor) [reply completion/available/grunt.completion.bash:5]: I have also tested grunt with [gulp](https://github.com/Bash-it/bash-it/pull/2137#discussion_r877858172) both are working on my end.

**Comment by tbhaxor** [comment]
Timeout

**Pull request closed without merging**